src/kbmod/reprojection.py

```diff
@@ -287,29 +281,26 @@ def _reproject_work_unit_in_parallel(
     # when all the multiprocessing has finished, convert the returned numpy arrays to RawImages.
     concurrent.futures.wait(future_reprojections, return_when=concurrent.futures.ALL_COMPLETED)
     time_stamps = []
```

Comment:
I think that we can remove this variable entirely, if we're able to use the `ImageStack.sort_by_time()` method.

Reply:
Cleaned it up